### DISCUSSION TITLE
Hungarian Translation

### DISCUSSION THREAD

**Discussion by kramerbala** [alfio-event/alf.io#1347] [translations]
Hi, we're interested in Hungarian translation, as native hungarian speakers. Currently Hungraian is not listed in POEditor, is there an option to list it?

**Comment by cbellone** [comment]
Hi,
thank you for your interest in translating alf.io to Hungarian!

I have added the Hungarian language to PoEditor

![image](https://github.com/alfio-event/alf.io/assets/3385346/5950d1c1-51a5-4bcb-9e43-163a730b7afb)

Please apply as a translator and I'll approve your request

Thank you!
